pkg/oc/admin/migrate/migrator.go

```diff
@@ -215,7 +215,8 @@ func (o *ResourceOptions) Complete(f *clientcmd.Factory, c *cobra.Command) error
 		break
 	}
 
-	o.Builder = f.NewBuilder(true).
+	o.Builder = f.NewBuilder().
+		Internal().
```

Comment:
revert this... we have to deal with unstructured objects in migrate

Reply:
or maybe make Unstructured()... not sure

Reply:
> revert this... we have to deal with unstructured objects in migrate

Not according to the old code.  Looks like it was always going into internal types.

Reply:
Pull open